packages/types/src/documents/app/automation.ts

```diff
@@ -10,6 +10,7 @@ export enum AutomationIOType {
   ARRAY = "array",
   JSON = "json",
   DATE = "date",
+  ATTACHMENT = "attachment",
```

Comment:
Shouldn't these strings be coming from the actual types?

Reply:
These types are automation input types - they've always been independent from the field types that our tables/columns use.

The types are similar, but not identical, for example automations have an `object` type - which can have custom types like row or table.

They are relatively independent from each other, as all these types are used for is informing the UI what form components to display for the various inputs/outputs of automation steps - they don't really tie up to Budibase data at all.